app/src/main/java/com/dozingcatsoftware/vectorpinball/util/MathUtils.java

```diff
@@ -69,7 +69,6 @@ public static float toRadiansF(float degrees) {
 
     public static float clamp(float x, float min, float max) {
         if (x < min) return min;
-        if (x > max) return max;
-        return x;
+        return Math.min(x, max);
```

Comment:
I prefer the original code for its symmetry. Although it would be better to rename the arguments to `lowerBound` and `upperBound` to avoid confusion with the min and max functions.